packer/ansible/roles/docker/README.md

```diff
@@ -1,5 +1,5 @@
 docker
-=========
+======
```

Comment:
It's attention to detail like this that keeps us on the CRITICAL PATH!

Reply:
# :boom: CRITICAL PATH :volcano: #